packages/paste-core/components/input/stories/input.stories.tsx

```diff
@@ -171,6 +171,35 @@ storiesOf('Components|Input', module)
       </>
     );
   })
+  .add('Input - Really Long Error', () => {
```

Comment:
Should we also add stories to Textarea, Select, and Combobox?

Reply:
I think this is enough